main1.tf

```diff
@@ -0,0 +1,39 @@
+provider "aws" {
+  region = "us-east-1"  
+}
+
+resource "aws_ecs_cluster" "example" {
+  name = "my-ecs-cluster"  # a name for your ECS cluster

```

Comment:
if possible mention a proper name instead of example and it would be easy that creating separe varibles file